src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -19,42 +18,50 @@ The following options can be configured on the root level of your browser-based
 | replaysSessionSampleRate | number | `0`     | The sample rate for replays that begin recording immediately and last the entirety of the user's session. `1.0` collects all replays, and `0` collects none.                                                                                                      |
 | replaysOnErrorSampleRate | number | `0`     | The sample rate for replays that are recorded when an error happens. This type of replay will record up to a minute of events prior to the error and continue recording until the session ends. `1.0` captures all sessions with an error, and `0` captures none. |
 
-The following options can be configured as options to the integration, in `new Replay({})`:
+The following can be configured as integration options in `new Replay({})`:
 
-| Key           | Type    | Default | Description                                                                                                                                                                        |
-| ------------- | ------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| stickySession | boolean | `true`  | Keep track of the user across page loads. Note, a single user using multiple tabs will result in multiple sessions. Closing a tab will result in the session being closed as well. |
+| Key                    | Type                 | Default | Description                                                                                                                                                                        |
+| ---------------------- | -------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| stickySession          | boolean              | `true`  | Keep track of users across page loads. Note, because closing a tab ends the session, a single user using multiple tabs will be recorded as multiple sessions. |
+| networkDetailAllowUrls | (string \| RegExp)[] | `[]`    | Capture request and response details for XHR and Fetch requests that match the given URLs.                                                                                                 |
```

Comment:
So the way this works is that `networkCaptureBodies` is on by default. So in reality you can use it to opt _out_ of capturing bodies.

Any URL matched by `networkDetailAllowUrls` will capture the bodies and the default headers (when no other config is set).

Reply:
@mydea "Any URL matched by networkDetailAllowUrls will capture the bodies and the default headers (when no other config is set)." --> Is this true for headers? 

Our "networkRequestHeaders" description says it's for ** capturing ** request headers for URLs defined in networkDetailAllowUrls, rather than "decide whether to capture.."